docs/quickstart/html.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Redoc HTML element
+---
+
+# ReDoc HTML element
```

Comment:
ReDoc or Redoc? It's spelled differently in the front matter - let's be consistent

Reply:
We intended to rename it to Redoc everywhere... (still to be done)

Reply:
Ah yeah - fixed those, should all be Redoc now.